File: Xamarin/SampleXamarin.iOS/Models/SensorStatusModel.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT license.

namespace SampleXamarin.iOS
{
    public interface SensorStatusModel
    {
        SensorStatus GeoLocationStatus { get; }
        SensorStatus WifiSignalStatus { get; }
        SensorStatus BluetoothSignalStatus { get; }
    }
}

File: Android/Java/app/src/main/java/com/microsoft/sampleandroid/SensorStatusModel.java
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT license.
package com.microsoft.sampleandroid;

public interface SensorStatusModel {
    SensorStatus getGeoLocationStatus();
    SensorStatus getWifiSignalStatus();
    SensorStatus getBluetoothSignalStatus();
}
